### ISSUE TITLE
RFC: using packages not in Project.toml

### ISSUE THREAD

**Issue by vtjnash** [JuliaLang/julia#49903]
I would frequently like to use packages that are in the Manifest.toml but not in the Project.toml from the REPL (for debugging, or getting Cthulhu to work). I can make it work sometimes with `using Foo.Bar` instead of simply `using Bar`. But I feel like I would really like to be able to use all Manifest packages interactively. Can we just make this work?

**Comment by KristofferC** [comment]
You can have multiple packages with the same name in the manifest so if you rely on this, your code can break if you e.g. update your packages. You could egive the UUID to disambiguate but that will probably be more annoying than just adding the package. 

I feel this should only be a convenience thing in e.g. the REPL.

**Comment by vtjnash** [comment]
Yes, only in the REPL (or `isinteractive`?), and only for unique names in the current Manifests, and only from toplevel modules (which don't have Project files)

**Comment by Seelengrab** [comment]
This could be a part of the "This package is available in a Registry" query you get in the REPL, right? So instead of just looking at the packages in the Project.toml, it can also look at the packages in Manifest.toml and suggest one of those if the names match.

**Comment by vtjnash** [comment]
Yes, but I emphatically don't want Pkg involved here. Sometimes that takes a long time to load and has to download stuff from the internet, which can be undesirable when the goal is to debug the current error and not introduce significant new stateful changes.

**Comment by Seelengrab** [comment]
It'd be a shortcut in the REPL frontend, before any expensive Pkg network requests. At least that's what I'm thinking of, so before even checking the registry, checking whether a package with that name is already available in the environment through some dependency chain.

Maybe it's just simpler in my head than in reality though :shrug:
